src/test/java/org/opentripplanner/graph_builder/module/StreetLinkerModuleTest.java

```diff
@@ -92,6 +102,37 @@ void linkFlexStop() {
     });
   }
 
+  @Test
+  void linkCarsAllowedStop() {
+    var model = new TestModel();
+    var carsAllowedTrip = TransitModelForTest
+      .of()
+      .trip("carsAllowedTrip")
+      .withCarsAllowed(CarAccess.ALLOWED)
+      .build();
+    model.withCarsAllowedTrip(carsAllowedTrip, model.stop());
+
+    var module = model.streetLinkerModule();
+
+    module.buildGraph();
+
+    assertTrue(model.stopVertex().isConnectedToGraph());
+
+    // Because the stop is used by a carsAllowed trip it needs to be linked to both the walk and car edge
+    assertEquals(2, model.stopVertex().getOutgoing().size());
```

Comment:
```suggestion
    assertThat(model.stopVertex().getOutgoing()).hasSize(2);
```

Since I wrote this other test, we introduced the truth assertion library.